### DISCUSSION TITLE
Way to close the ScriptKit modal window (or wait for it to be closed) after the script is selected?

### DISCUSSION THREAD

**Discussion by ramiroaraujo** [johnlindquist/kit#1195] [Q&A] [answered]
I need to trigger a screenshot in mac with `/usr/sbin/screencapture`, but the modal get's in the way :P. Any idea how to close/hide it?
Thanks!

**Comment by johnlindquist** [comment, marked as answer]
@ramiroaraujo Try this:

```js
await hide()
```

**Reply by ramiroaraujo** [reply]
Thanks! ChatGTP already found it for me 🤣. It even proposed a `show` that doesn't exist to call after the screenshot for some reason

**Reply by ramiroaraujo** [reply]
oh, it does exists :D
